react-native-libraries.json

```
@@ -9518,5 +9518,19 @@
     "ios": true,
     "android": true,
     "dev": true
+  },
+  {
+    "githubUrl": "https://github.com/bambuser/react-native-bambuser-broadcaster",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "unmaintained": true,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```

Thank you for the explanation and checks! 👍  If the library works with the latest version of React Native it is fine to add it, besides the `unmaintained` flag I think, so LGTM.

Let's just clean up the entry, before we merge that in:
```suggestion
    "android": true,
    "unmaintained": true
```

---

It is done as suggested :)